docs/hub/_toctree.yml

```diff
@@ -125,12 +125,33 @@
     title: Dataset Cards
   - local: datasets-gated
     title: Gated Datasets
+  - local: datasets-adding
+    title: Adding a new Dataset
   - local: datasets-viewer
     title: Dataset Viewer
-  - local: datasets-usage
-    title: Using Datasets
-  - local: datasets-adding
-    title: Adding New Datasets
+    sections:
+      - local: datasets-viewer-configure
+        title: Configure the Dataset Viewer
+  - local: datasets-data-files-configuration
+    title: Data files Configuration
+    sections:
+      - local: datasets-file-names-and-splits
+        title: File names and splits
+      - local: datasets-manual-configuration
+        title: Manual Configuration
```

Comment:
Multiple splits and subsets is a bit more advanced use case, so instead of copying this info from the `datasets` docs, I think it's better to have a section (or FAQ) in "Upload Datasets"/"Download Datasets" that links to the `datasets` docs. And we can have examples of loading multiple splits and subsets using `datasets`/`datasets-server` (to inspect such repos) in the "Integrated Libraries" sections.

Plus, it would be nice to explain the reasoning behind sharded uploads (can bypass the Hub's file size limit, incremental uploads, etc.) with some examples of loading such datasets using `fsspec`'s `glob` (in the "Integrated Libraries" part ofc)

Reply:
> Multiple splits and subsets is a bit more advanced use case, so instead of copying this info from the datasets docs, I think it's better to have a section (or FAQ) in "Upload Datasets"/"Download Datasets" that links to the datasets docs.

I think we better have Hub documentation that doesn't depend on the `datasets` lib - people don't need to know about it to share a dataset on the Hub.

> And we can have examples of loading multiple splits and subsets using datasets/datasets-server (to inspect such repos) in the "Integrated Libraries" sections.

It would be cool indeed. I left the integrated libraries docs quite bare-bone for this PR on purpose, we can enrich them later

> Plus, it would be nice to explain the reasoning behind sharded uploads (can bypass the Hub's file size limit, incremental uploads, etc.) with some examples of loading such datasets using fsspec's glob (in the "Integrated Libraries" part ofc)

Yup !

Reply:
Makes sense! Maybe we can have a link to the "Data files Configuration" page in "Uploading Datasets". This would probably be better than accessing this info through the "Configure the Viewer" page. Only the `datasets` lib and the viewer understand these configurations, so I guess the current situation is also fine.

Reply:
I added a link in the new section about supported file formats in the "Uploading Datasets" page